.github/workflows/ci.yml

```diff
@@ -19,21 +19,18 @@ jobs:
     steps:
       - name: Checkout cedar-spec
         uses: actions/checkout@v3
-      - name: Download Dafny
+      - name: Install Z3
         shell: bash
-        run: wget https://github.com/dafny-lang/dafny/releases/download/v4.0.0/dafny-4.0.0-x64-ubuntu-20.04.zip
```

Comment:
just curious how we get Dafny now (and which version) since I don't see a direct replacement for this line?

Reply:
Dafny is installed via dotnet tool. I think GitHub Action Ubuntu images have dotnet preinstalled.